control/table.go

```diff
@@ -71,20 +62,41 @@ func (ct *Table) controlName() (name string, err error) {
 
 // Fill inserts the OpenControl justifications into the table. Note this modifies the `table`.
 func (ct *Table) Fill(openControlData opencontrols.Data) (err error) {
-	roleCell, err := ct.responsibleRoleCell()
+	roleCell, err := findResponsibleRole(ct)
 	if err != nil {
 		return
 	}
 
-	existingContent := roleCell.Content()
 	control, err := ct.controlName()
 	if err != nil {
 		return
 	}
 
 	roles := openControlData.GetResponsibleRoles(control)
-	content := fmt.Sprintf("%s %s", existingContent, roles)
-	roleCell.SetContent(content)
+	roleCell.setValue(roles)
 
 	return
 }
+
+// diffResponsibleRole computes the diff of the responsible role cell.
+func (ct *Table) diffResponsibleRole(control string, openControlData opencontrols.Data) ([]reporter.Reporter, error) {
+	roleCell, err := findResponsibleRole(ct)
+	if err != nil {
+		return []reporter.Reporter{}, err
+	}
+	yamlRoles := openControlData.GetResponsibleRoles(control)
+	sspRoles := roleCell.getValue()
+	if roleCell.isDefaultValue(sspRoles) || yamlRoles == sspRoles {
```

Comment:
definitely, i agree. `getValue` for `sspValue  trims whitespaces but `yamlRoles` is a but unsure how it will be. when we get away from using `FormatResponsibleRole` in `GetResponsibleRole` in masonry, we can address that then